apps/playground/src/app/modules/components/error-page-module/error-page-module.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Angular imports
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+/**
+ * Internal imports
+ */
+import { ErrorPageModuleComponent } from './error-page-module.component';
+
+describe('ErrorPageModuleComponent', () => {
+  let component: ErrorPageModuleComponent;
+  let fixture: ComponentFixture<ErrorPageModuleComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ErrorPageModuleComponent]
```

Comment:
I have mistakes when I run it

Reply:
which ones ? it works for me

Reply:
Nevermind, it's not that important for the infranum app and we need to move on anyway...